.github/workflows/test.yml

```diff
@@ -32,6 +32,11 @@ jobs:
         - python-version: "3.10"
           toxenv: "asyncio"
 
+        - python-version: "3.11"
+          toxenv: "py"
+        - python-version: "3.11"
+          toxenv: "asyncio"
```

Comment:
Should we remove 3.10 asyncio env?

Reply:
Good call. Synced it up with scrapy's CI setup: https://github.com/scrapy/scrapy/blob/master/.github/workflows/tests-ubuntu.yml#L17-L25